src/Storefront/Resources/app/storefront/src/scss/abstract/variables/_bootstrap.scss

```
@@ -3,7 +3,7 @@ Bootstrap variables
 ==================================================
 Contains all overrides of bootstrap default variables.
 
-https://getbootstrap.com/docs/4.3/getting-started/theming/#variable-defaults
+https://getbootstrap.com/docs/5.0/customize/sass/#variable-defaults
```

Currently the default bootstrap version is still v4, v5 will only be used if you activate the feature flag for the next major version. Bootstrap v5 will be the default starting with v6.5.0.0.
What do you think of adding both documentation links:

```suggestion
Bootstrap 4 Docs
https://getbootstrap.com/docs/4.3/getting-started/theming/#variable-defaults
Bootstrap 5 Docs
https://getbootstrap.com/docs/5.0/customize/sass/#variable-defaults
```